src/components/MainComponent.js

```diff
@@ -17,6 +17,26 @@ import Debug from 'components/Debug'
 import { isChromeExtensionBackground } from 'services/notificationService'
 import Notifications from 'components/Extension/Notifications'
 
+try {
+  const Checklist = require('components/Checklist')
+}
+catch(error) {
+  if ( !error.toString().includes('Cannot find module') ) {
+    throw error
+  } else {
+
+  }
+}
+
+const isPluginDefined = () => {
```

Comment:
This should also work as 

    const isPluginDefined = () => (typeof Checklist !== undefined)

Actually, I'm now slightly confused, because the above line returns the opposite of the current version. isPluginDefined returns true when Checklist is undefined?